spec/unit/provider/package/gem_spec.rb

```diff
@@ -36,6 +40,32 @@
         allow(provider).to receive(:rubygem_version).and_return('1.9.9')
       end
 
+      context 'on windows' do
+        let (:path) do
+          "C:\\Program Files\\Puppet Labs\\Puppet\\puppet\\bin;C:\\Program Files\\Puppet Labs\\Puppet\\bin;C:\\Ruby26-x64\\bin;C:\\Windows\\system32\\bin"
+        end
+        let(:expected_path) do
+          "C:\\Program Files\\Puppet Labs\\Puppet\\bin;C:\\Ruby26-x64\\bin;C:\\Windows\\system32\\bin"
+        end
+        before do
+          allow(Puppet::Util::Platform).to receive(:windows?).and_return(true)
+          allow(Puppet::Util).to receive(:get_env)
+          allow(Puppet::Util).to receive(:get_env).with('PATH').and_return(path)
+          allow(described_class).to receive(:validate_command).with(provider_gem_cmd)
+          stub_const('::File::PATH_SEPARATOR', ';')
+        end
```

Comment:
nit: would add some empty lines between let/let/before for consistency; also you have a whitespace in the first `let (:path)`